### DISCUSSION TITLE
focus after error when using quasar internal validation with  q-field

### DISCUSSION THREAD

**Discussion by JulienDeBerlin** [quasarframework/quasar#10197] [General - Components / Directives / etc]
I'm setting up a form with quasar and use the internal validation. I have a group of checkboxes. The user should select at least one of them. I've managed the validation on form submit, but the only thing that is annoying me: if the validation fails (aka the user didn't select any checkbox), then there is no focus on that field as it is the case for all other fields I use the internal validation for. I guess it has something to do with me using a q-field (instead of q-input).

Here is my code:
```
<q-field v-model="selectedFruits"
         ref="myFruits"
         class="q-gutter-sm"
         :rules="[ val => val.length >= 1 || 'Select at least one fruit']"
         :disable="isReadOnly()">
                <q-checkbox v-model="selectedFruits" val="Apple" label="Apple"/>
                <q-checkbox v-model="selectedFruits" val="Banana" label="Banana"/>
                <q-checkbox v-model="selectedFruits" val="Pear" label="Pear"/>
                <q-checkbox v-model="selectedFruits" val="Peach" label="Peach"/>
</q-field>
```

When I click on submit, the validation is done, but without putting the focus on the q-field, which is not user-friendly at all. How could I fix this?

**Comment by ontwikkelfabriek** [comment]
@JulienDeBerlin 

For better support , please create a codepen.io to demo your issue.

**Comment by hhosseinipaymentshield** [comment]
@JulienDeBerlin did you fix your issue? I'm having the same problem
